src/app/pages/news/11.2/08-market.md

```diff
@@ -1,10 +1,33 @@
-## New Market artifacts {#market}
+## Growing Market {#market}
+
+Our Market is growing and offering more goods from the shelf.
+These products are compatible with LTS 10 too, so there's no need to join the leading-edge train in order to use these powerful products in your daily work.
+
+- **AI and Robotics**: let's delegate the tedious work to the machines!
+  - [Axon Ivy RPA](https://market.axonivy.com/metaproc-connector)
+  - [Chat GPT Connector](https://market.axonivy.com/openai-connector)
+  - [DeepL Connector](https://market.axonivy.com/deepl-connector)
+- **Swissness**: local goods built in our headquarter
```

Comment:
headquarter -> headquarters (Im Englishen immer Mehrzahl)

Reply:
every day I learn something new: thanks for sharing :100: